default/scripting/species/SP_ANCIENT_GUARDIANS.focs.txt

```diff
@@ -0,0 +1,81 @@
+Species
+    name = "SP_ANCIENT_GUARDIANS"
+    description = "SP_ANCIENT_GUARDIANS_DESC"
+    gameplay_description = "SP_ANCIENT_GUARDIANS_GAMEPLAY_DESC"
+
+    tags = [ "ROBOTIC" "AI_TAG_NO_INDUSTRY" "AI_TAG_NO_RESEARCH" "AI_TAG_AVERAGE_SUPPLY"]
+
+    foci = [
+        [[HAS_INDUSTRY_FOCUS]]
+        [[HAS_RESEARCH_FOCUS]]
+        [[HAS_GROWTH_FOCUS]]
+        [[HAS_MINING_FOCUS]]
```

Comment:
Does it make sense for a non-playable, non-conquerable species to have any foci?